projects_info.csv

```diff
@@ -48,7 +48,7 @@ Y | actinia | 4.2.0 | Y | Y | Web Services | | Processing of Geographical Data |
 Y | eoxserver | 1.1.2 | Y | Y | Web Services | | Web Coverage Service | eoxserver | https://docs.eoxserver.org/en/stable/ | | EOxServer | Y
 Y | ETF | 2.0.0 | Y | Y | Web Services | | Testing framework for validating spatial data, metadata and web services in Spatial Data Infrastructures (SDIs) | etf-validator | https://github.com/etf-validator| | ETF | N
 Y | qgis_mapserver | 3.34.9 | Y | Y | Web Services | | Web Map Service | | https://www.qgis.org/ | | QGIS Server | Y
-Y | re3gistry | 2.4.2 | Y | Y | Web Services | | Management of reference codes | re3gistry | https://github.com/ec-jrc/re3gistry | | Re3gistry | N
+Y | re3gistry | 2.6.1 | Y | Y | Web Services | | Management of reference codes | re3gistry | https://github.com/ec-jrc/re3gistry | | Re3gistry | N
```

Comment:
@kalxas  Is this the installed version on OSGeoLive 17?

Reply:
We have not yet installed, since it is VM only, so I would accept it now.